src/content/docs/new-relic-solutions/best-practices-guides/alerts-applied-intelligence/alerts-best-practices.mdx

```diff
@@ -22,174 +22,217 @@ redirects:
   - /docs/new-relic-solutions/new-relic-solutions/optimize-your-cloud-native-environment/set-proactive-alerts-align-teams-tools-processes-incident-response/
 ---
 
-Improve your alerts coverage by implementing the following recommendations and get the most out of your alerts configuration.
+New Relic enables you to create and customize your own alerts. You can improve your alerts coverage by implementing the following recommendations and get the most out of your alerts configuration.
```

Comment:
We like to avoid this phrase: "New Relic [verbs]...."

Years back we had a sweep of the docs site where we tried to delete as many mentions of "New Relic" as possible on the docs site.

Reply:
Thank you, @paperclypse! I've modified it.